### PR TITLE
[DOC release] Documentation changes

### PR DESCRIPTION
**Pull request by adityabhatia52**
Documentation has been modified according to #13862

### CODE REVIEW

**Comment by rwjblue** [comment]
Can you update to remove the documentation that this replaces?

File: packages/ember-testing/lib/helpers/fill_in.js [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+  Example:
+  ```javascript
+  var $el = find('.my-selector');
+  ```
+  @method find
+  @param {String} selector jQuery string selector for element lookup
+  @return {Object} jQuery object representing the results of the query
+  @public
+*/
+
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/fill_in.js]: Can you delete this extra line?

**Comment by rwjblue** [comment]
Can you add the following snippet to the top of each helper file:

```js
/**
@module ember
@submodule ember-testing
*/
```

Thanks!

File: packages/ember-testing/lib/helpers/fill_in.js [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+  for `app.$(selector)`.
+  Example:
+  ```javascript
+  var $el = find('.my-selector');
+  ```
+  @method find
+  @param {String} selector jQuery string selector for element lookup
+  @return {Object} jQuery object representing the results of the query
+  @public
+*/
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/fill_in.js]: Can you remove this blank line?

File: packages/ember-testing/lib/helpers/fill_in.js [hunk 3]
```diff
@@ -1,4 +1,20 @@
+/**
+@module ember
+@submodule ember-testing
+*/
 import { focus, fireEvent } from '../events';
+/**
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/fill_in.js]: Can you add a blank line above this one?

**Comment by rwjblue** [comment]
Looks great! Would you mind squashing the commits?

**Comment by adityabhatia52** [comment]
I think it is done!

**Comment by rwjblue** [comment]
Looks like this is including other changes done on master since the PR was proposed. Would you mind cleaning it up?

**Comment by adityabhatia52** [comment]
I am not sure on how to do that?? Would you please guide me on how to squash commits without including other changes?

**Comment by mixonic** [comment]
Closed in https://github.com/emberjs/ember.js/pull/14107 - Thanks!

**Pull request closed without merging**